cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/tf_multi_exec.py

```diff
@@ -0,0 +1,215 @@
+import argparse
+import configparser
+import logging
+import os
+import sys
+from dataclasses import dataclass
+from hashlib import blake2b
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+from az.cli import az
+from python_terraform import IsFlagged, IsNotFlagged, Terraform
+
+log = logging.getLogger(__name__)
+logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.WARNING)
+EX_FAILED: int = 1  # exit  code for failed command
+PROFILES_FILE: str = "profiles.ini"
+
+
+@dataclass
+class AzureProfile:
```

Comment:
Considering that the code is Azure specific, we should call it `azure_tf_multi_exec.py` (and yes, the AWS version should be `aws_tf_multi_exec.py`). ... just in case a customer was using both AWS and Azure :).

Reply:
Maybe we can call the scripts azure_kentik_onboarder.py and aws_kentik_onboarder.py ? Any ideas for a better name?

Reply:
I like the `*_onboarder.py` names.